server/src/repository/ReviewsDAO.js

```diff
@@ -0,0 +1,34 @@
+const { DynamoDBClient } = require('@aws-sdk/client-dynamodb');
+const { fromIni } = require('@aws-sdk/credential-provider-ini');
+const { DynamoDBDocumentClient, PutCommand, ScanCommand, UpdateCommand, GetCommand } = require('@aws-sdk/lib-dynamodb');
+const { logger } = require('../util/logger');
+
+const client = new DynamoDBClient({
+    region: "us-west-1",
+    credentials: fromIni({profile: "cb_account"})
+});
+
+const documentClient = DynamoDBDocumentClient.from(client);
+
+const TableName = "CookBooked-Reviews";
+
+//POST the new review
```

Comment:
If you want, you can use
    /**
    *
    */ 
and put a better doc comment.